src/blog/2025-09-19-oss-summit-europe-2025.md

```diff
@@ -0,0 +1,101 @@
+
+Tie programs to these measures. Show the path from activity to outcome. For example, a tutorial refresh is not the outcome. The outcome is a lower time to first value for the target persona, confirmed by product analytics and user feedback. A community office hours session is not the outcome. The outcome is fewer stalled evaluations and more developers reaching activation in the same quarter.
+
+When you have outcomes, ask for resources to scale the programs that work. Use the business language your leadership uses. You are not asking for a headcount. You are proposing an investment that increases activation by a defined amount at a known cost.
+
+### How do we tie DevRel to revenue?
+
+Revenue is an outcome of successful journeys. DevRel can shorten and strengthen those journeys. The work is to trace developer progress across the community and the product.
+
+* **Connect the dots.** Use community intelligence and product analytics to see who is evaluating, who is unblocked, and who is converting. You do not need perfect attribution. You need a consistent way to see movement across key stages.  
+* **Instrument the handoffs.** Track how DevRel content and support reduce common blockers in proofs of concept. Track time from the first question to unblock. Track how often a successful proof of concept moves to a trial or purchase.  
+* **Capture stories and systematize them.** When you help a team move from evaluation to purchase faster, write the play. Which persona? Which blockers? Which content or support solved it? Turn that into a repeatable motion that anyone in your company can use.
+
+The Metrics working group is rebooting to publish examples you can adapt. Expect templates for funnel definitions, example dashboards, and sample reports that map DevRel outcomes to pipeline stages and customer health.
+
+### What are the most common challenges?
+
+We heard three over and over again.
+
+* **Perception problem.** DevRel is considered a cost center without clear value. The remedy is consistent language, transparent goals, and outcome reports that map to company OKRs.  
+* **Silos and access.** Teams do not have access to product analytics or cannot connect community signals to product data. The remedy is cross-functional alignment and a minimum shared instrumentation plan. Start small. Agree on the few events and labels everyone will use.  
+* **Unsustainable motions.** Travel-heavy schedules burn people out and waste money. Replace a portion of travel with scalable digital programs, local community partners, and content that compounds over time. When you do travel, tie every trip to a measurable goal.
+
+The Foundation will publish playbooks to facilitate cross-functional collaboration and promote sustainable operating models. These will include checklists, templates for shared dashboards, and example rhythms for data reviews involving product, marketing, and sales teams.
```

Comment:
The foundation is working on a [DevRel Maturity Model](https://github.com/DevRel-Foundation/wg-resource-aggregation?tab=readme-ov-file#working-group-projects) to publish playbooks for facilitating cross-function collaboration and promote...